### PR TITLE
Configure fix

### PR DESCRIPTION
**Pull request by tloetzer**
Updated @yannik1015 's work to work with the latest state. I have also confirmed that it works with my goe charger.

### CODE REVIEW

File: custom_components/goecharger/translations/en.json
Comment (mpw96) [inline custom_components/goecharger/translations/en.json:1]: Please make the same changes also in strings.json.

**Review by mpw96 — CHANGES_REQUESTED** [review]

File: custom_components/goecharger/translations/en.json
Comment (tloetzer) [reply custom_components/goecharger/translations/en.json:1]: done

File: custom_components/goecharger/config_flow.py [hunk 1]
```diff
@@ -42,3 +44,40 @@ async def async_step_user(self, info):
...
+                    vol.Required(CONF_HOST, default=current_host): str,
+                    vol.Optional(
+                        CONF_SCAN_INTERVAL, default=current_scan_interval if current_scan_interval else 20 
+                    ): int,
+                    vol.Required(
+                        CONF_CORRECTION_FACTOR, default=current_correction_factor if current_correction_factor else "1.0"
+                    ): str,
+                }
+            )
+        )
```
Comment (mpw96) [inline custom_components/goecharger/config_flow.py]: Please add the missing newline.

**Review by mpw96 — CHANGES_REQUESTED** [review]

**Review by mpw96 — APPROVED** [review]
LGTM

**Comment by mpw96** [comment]
hassfest failure is a known issue.

**Pull request merged**